sourcecode/apis/contentauthor/tests/Integration/Libraries/H5P/API/H5PImportControllerTest.php

```diff
@@ -31,12 +30,15 @@ function is_uploaded_file($filename)
     use Illuminate\Support\Facades\Storage;
     use Tests\Helpers\MockVersioningTrait;
     use Tests\TestCase;
-    use function base_path;
+
+use function base_path;
```

Comment:
Indentation

Reply:
There's a bug in php-cs-fixer that causes this to always be re-added. I'd like to remove the function override that's used in this file, which will side-step the issue, otherwise I don't think it's worth blocking over.